seednode/docker/prod/Dockerfile

```diff
@@ -1,16 +1,27 @@
-FROM openjdk:8-jdk
-
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    openjfx && rm -rf /var/lib/apt/lists/*
-
-WORKDIR /bisq-seednode
-CMD ./docker/startSeedNode.sh
-
-ENV APP_NAME=seednode
-ENV NODE_PORT=8000
+# first the builder builds the executables
+###
+FROM gradle:jdk11 as builder
+ARG SEEDNODE_URL
+ARG SEEDNODE_BRANCH
+RUN git clone $SEEDNODE_URL
```

Comment:
This is already in the same repository as the seednode code, we can use local data instead of fetching it from github